src/proposals/20250807/SparkEthereum_20250807.t.sol

```diff
@@ -0,0 +1,245 @@
+
+import { InterestStrategyValues, ReserveConfig } from '../../test-harness/ProtocolV3TestBase.sol';
+import { ICustomIRM, IRateSource } from '../../test-harness/SparkEthereumTests.sol';
+import { SparkLendContext }        from '../../test-harness/SparklendTests.sol';
+import { SparkTestBase }           from '../../test-harness/SparkTestBase.sol';
+
+contract SparkEthereum_20250807Test is SparkTestBase {
+
+    address internal constant PYUSD               = 0x6c3ea9036406852006290770BEdFcAbA0e23A0e8;
+    address internal constant PYUSD_PRICE_FEED    = 0x42a03F81dd8A1cEcD746dc262e4d1CD9fD39F777;
+    address internal constant USDS_DAI_IRM        = 0xD99f41B22BBb4af36ae452Bf0Cc3aA07ce91bD66;
+    address internal constant USDS_DAI_IRM_OLD    = 0xE15718d48E2C56b65aAB61f1607A5c096e9204f1;
+    address internal constant USDC_USDT_IRM_OLD   = 0x7F2fc6A7E3b3c658A84999b26ad2013C4Dc87061;
+    address internal constant USDC_USDT_PYUSD_IRM = 0xD3d3BcD8cC1D3d0676Da13F7Fc095497329EC683;
+
+    constructor() {
+        id = "20250807";
+    }
+
+    function setUp() public {
+        setupDomains("2025-07-29T15:43:00Z");
+
+        deployPayloads();
+
+        // chainData[ChainIdUtils.Ethereum()].payload = 0x41EdbF09cd2f272175c7fACB857B767859543D15;
+
+        // Deal the amount of pyUSD to Spark Proxy
+        deal(PYUSD, Ethereum.SPARK_PROXY, 1e6);
+    }
+
+    function test_ETHEREUM_sparkLend_collateralOnboarding() public onChain(ChainIdUtils.Ethereum()) {
+        SparkLendAssetOnboardingParams memory params = SparkLendAssetOnboardingParams({
+            // General
+            symbol:            'PYUSD',
+            tokenAddress:      PYUSD,
+            oracleAddress:     PYUSD_PRICE_FEED,
+            collateralEnabled: false,
+            // IRM Params
+            optimalUsageRatio:      0.95e27,
+            baseVariableBorrowRate: 0,
+            variableRateSlope1:     0.015e27,
+            variableRateSlope2:     0.15e27,
+            // Borrowing configuration
+            borrowEnabled:          true,
+            stableBorrowEnabled:    false,
+            isolationBorrowEnabled: false,
+            siloedBorrowEnabled:    false,
+            flashloanEnabled:       true,
+            // Reserve configuration
+            ltv:                  0,
+            liquidationThreshold: 0,
+            liquidationBonus:     0,
+            reserveFactor:        10_00,
+            // Supply caps
+            supplyCap:    50_000_000,
+            supplyCapMax: 500_000_000,
+            supplyCapGap: 50_000_000,
+            supplyCapTtl: 12 hours,
+            // Borrow caps
+            borrowCap:    25_000_000,
+            borrowCapMax: 475_000_000,
+            borrowCapGap: 25_000_000,
+            borrowCapTtl: 12 hours,
+            // Isolation  and emode configurations
+            isolationMode:            false,
+            isolationModeDebtCeiling: 0,
+            liquidationProtocolFee:   0,
```

Comment:
10%

Reply:
this does not get set in AaveV3ConfigEngine when ltv = 0